linkerd/app/src/env.rs

```diff
@@ -500,6 +520,51 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
     })
 }
 
+fn oc_trace_labels(oc_file_path: Result<Option<String>, EnvError>) -> HashMap<String, String> {
+    let oc_labels = match oc_file_path {
+        Ok(path_option) => match path_option {
+            Some(path) => match fs::read_to_string(path.clone()) {
+                Ok(label_string) => convert_labels_to_map(label_string),
+                Err(err) => {
+                    warn!(
+                        "could not read OC trace labels file at path {}: {}",
+                        path, err
+                    );
+                    HashMap::<String, String>::new()
+                }
+            },
+            None => {
+                warn!("env variable {} not present", ENV_LABELS_FILE_PATH);
+                HashMap::<String, String>::new()
+            }
```

Comment:
Do we expect this env var to always be set? If it's only set when OpenCensus is in use, I am not sure if this should be a warning.

Reply:
Hmm, removed the warning! but still working on https://github.com/linkerd/linkerd2/pull/4199, to make the `env`, `volumeMounts` present only when tracing is enabled. (which are present by default before)